src/js/components/FocusedContainer.js

```diff
@@ -23,12 +23,14 @@ export class FocusedContainer extends Component {
   ref = React.createRef();
 
   componentDidMount() {
-    const { hidden } = this.props;
+    /* eslint-disable react/destructuring-assignment */
+    const ariaHidden = this.props['aria-hidden'];
```

Comment:
Can you include this link in a comment? https://reactjs.org/docs/accessibility.html#wai-aria

Reply:
can we keep `hidden` and do `aria-hidden={hidden}` inside this component ?

Reply:
I would add `aria-hidden={hidden}` in line 69